src/pages/explorePage/ExplorePage.tsx

```diff
@@ -99,6 +138,41 @@ const ExplorePageContent = ({
     hasPrereqs,
   };
 
+  const mapFilterStateToURL = (fs: SearchFilterState) => {
+    return {
+      exclude: fs.courseCodes
+        .map((bool, index) => (bool ? null : index))
+        .filter((index) => index !== null),
+      numCourseRatings: !fs.numCourseRatings ? null : fs.numCourseRatings,
```

Comment:
can we rename to minCourseRatings and minProfRatings to be more explicit

Reply:
For sure. Do you think we should define a custom interface for representing the filterState in the URL at this rate actually?

Reply:
Okay added interface.